docs/docs/associations.md

```diff
@@ -75,12 +77,14 @@ User.belongsTo(Company, {foreignKey: 'fk_companyname', targetKey: 'name'}); // A
 HasOne associations are associations where the foreign key for the one-to-one relation exists on the **target model**.
 
 ```js
-var User = sequelize.define('User', {/* ... */})
-var Project = sequelize.define('Project', {/* ... */})
+var User = sequelize.define('user', {/* ... */})
+var Project = sequelize.define('project', {/* ... */})
  
 // One-way associations
 Project.hasOne(User)
- 
+
+Observe Project is **source model** and User is **target model**. In `hasOne` User have information about its Project as `ProjectId`, information exists on **target model**. This relation will add a `ProjectId` attribute to **target model** i.e. User model.
+
```

Comment:
Not sure this is the most ideal explanation, we might need a section "Difference between hasOne and belongsTo"